fern/docs/pages/manual/ingestion.mdx

```diff
@@ -34,4 +34,27 @@ deleting all contents of `local_data` folder (except .gitignore).
 To simplify this process, you can use the command:
 ```bash
 make wipe
-```
\ No newline at end of file
+```
+
+## Supported file formats
+All the files in the following list are supported.
+
+* ".hwp"
+* ".pdf"
+* ".docx"
+* ".pptx"
+* ".ppt"
+* ".pptm"
+* ".jpg"
+* ".png"
+* ".jpeg"
+* ".mp3"
+* ".mp4"
+* ".csv"
+* ".epub"
+* ".md"
+* ".mbox"
+* ".ipynb"
+* ".json"
+
+Other file formats might work, but they will be considered as plain text files (in other words, they will be ingested as `.txt` files).
```

Comment:
We could also mention that not all doc format loaders are installed by default. if you try to ingest a special format  PGPT may fail with an explanatory error asking to install an extra dependency to be able to ingest that format. Install it following that error suggestion and try again.

(I'd say it is the case for .epub for example)